windows/install_net35_20h2.bat

```diff
@@ -0,0 +1,21 @@
+::
+:: Installs .NET Runtime 3.5, used by Wix 3.11 and the Visual C++ Compiler for Python 2.7
+:: Taken from the mcr.microsoft.com/dotnet/framework/runtime:3.5 Dockerfile:
+;; https://github.com/microsoft/dotnet-framework-docker/blob/e0b59f4aeb47bd6bf13e4c7ec6676a1935306df9/src/runtime/3.5/windowsservercore-20H2/Dockerfile
```

Comment:
This comment should start with `::`. This line currently causes an error:
```
C:\>https://github.com/microsoft/dotnet-framework-docker/blob/e0b59f4aeb47bd6bf13e4c7ec6676a1935306df9/src/runtime/3.5/windowsservercore-20H2/Dockerfile
'https:' is not recognized as an internal or external command,
operable program or batch file.
```

(which doesn't make the script fail, since we don't exit on error here)